README.md

```diff
@@ -1,30 +1,57 @@
-# SourceExplorer
-Last updated by | Egor Belekhov | Nov 23, 2021 at 11:23 AM GMT+3
+# GitRunner
+Last updated by | Egor Belekhov | Nov 24, 2021 at 10:03 AM GMT+3
 ***
 
-## Overview
+# GitRunner
 
-We need an application that will reduce the time and simplify the work with git. It should provide the ability to manage all the necessary repositories from one place (change, update branches) as well as build and run the necessary parts of the project.
+## Overview
 
+We need an application that will reduce the time and simplify the work with git. It should provide the ability to manage all the necessary repositories from one place (change, update branches).
 
 ### Use Cases
 
-SourceExplorer should be user to update, build and run microservices. The list of paths to the necessary repositories will be in the configuration file.
-For each of them, it will be possible to see the included branch and change it to another existing branch.  The selected branch can be updated by performing a pull operation. 
-The next option is to select a mod and build the project according to it.
-And lastly launching the project. Also these possibilities should be available for a group of projects, i.e. the selection of settings and then the execution of the necessary logic for all of them.
+The tool should be able to:
+
+1. Synchronize repositories (all or selected). Pull the branch that is currently selected. Selected branch shall be visible to user.
+2. Have the ability to change between schemas - change multiple repositories to 9.0 or 10.0 (master). Synchronization is considered a separate operation.
+3. Have an autostash feature - be able to stash changes for current branch before pulling.
+4. Have a yaml configuration file in a repo to get the configuration from.
+5. Automatically prune branches that do not have remote branches.
+6. Have the ability to manually add repositories for synchronization from FossConnect project.
+7. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated 'View logs' button to open log folder for convinience.
```

Comment:
It seems that we would also need a 'View config' button that will open the folder with the configuration - because if we will deploy the application via ClickOnce it will be very hard to find it in the file system.

Reply:
added info about second button